modules/module8/slides/module8_05.qmd

```diff
@@ -0,0 +1,269 @@
+---
+format: revealjs
+title: Coefficients and coef_
+title-slide-attributes:
+  data-notes: |
+---
+
+```{python}
+#  | echo: false
+%run src/utils.py
+```
+
+## Intuition behind linear regression
+
+
+| listing number | Number of Bedrooms | Number of Bathrooms | Square Footage | Age | Price |
+|----------------|--------------------|---------------------|----------------|-----|-------| 
+| 1              | 5                  | 6                   | 3000           | 2   | $6.39 million|
+| 2              | 1                  | 1                   | 800            | 90  | $1.67 million|
+| 3              | 3                  | 2                   | 1875           | 66  | $3.92 million|
+: {tbl-colwidths="[16,16,16,16,10,16]"}
+
+:::{.notes}
+Unlike with decision trees where we make predictions with rules and analogy-based models where we predict a certain class using distance to other examples, linear classifiers use **coefficients** (or sometimes known as "weights") associated with features.
+:::
+
+---
+
+![](../../../static/module8/house_table.png){fig-align="center" width="50%" fig-alt="404 image"}
+
+
+:::{.notes}
+We then use these learned coefficients to make predictions.
+
+For example, suppose we are predicting the price of a house and we have 4 features;  number of bedrooms, number of bathrooms, square footage, and age.
+:::
+
+---
+
+![](../../../static/module8/house_table.png){fig-align="center" width="35%" fig-alt="404 image"}
+
+<br>
+
+Consider the following listing (example):     
+
+| listing number | Number of Bedrooms | Number of Bathrooms | Square Footage | Age |
+|----------------|--------------------|---------------------|----------------|-----| 
+| 3              | 3                  | 2                   | 1875           | 66  |
+
+
+&nbsp;&nbsp;
+
+<font size="4"><em> predicted(price) = coefficient<sub>bedrooms</sub> x #bedrooms + coefficient<sub>bathrooms</sub> x #bathrooms + coefficient<sub>sqfeet</sub> x #sqfeet + coefficient<sub>age</sub> x age + intercept </em></font>
+
+
+<font size="4"><em> predicted(price) = 0.03 x #bedrooms + 0.04 x #bathrooms + 0.002 x #sqfeet + -0.01 x #age + intercept </em></font>
+
+
+<font size="4"><em> predicted(price) = (0.03 x 3) + (0.04 x 2) + (0.002 x 1875) + (-0.01 x 66) + 0 </em></font>
+
+<font size="4"><em> predicted(price) = 3.26 </em></font>
```

Comment:
This currently flows together in a way that makes it hard to read:

<img width="1132" height="478" alt="image" src="https://github.com/user-attachments/assets/76cc3375-c5e7-492c-9947-bacdb3661877" />

Could you try either adding `<br>` tags to separate the lines or maybe even formatting as math with surrounding `$`?